pkgs/racket-doc/scribblings/reference/stx-trans.scrbl

```diff
@@ -262,7 +262,7 @@ The @racket[stop-ids] argument controls how far @racket[local-expand] expands @r
 
  @item{If @racket[stop-ids] is any other list, then expansion proceeds recursively until the expander
        encounters any of the forms in @racket[stop-ids], and the result is the partially-expanded
-       form. If @racket[extend-stop-ids?] is not @racket[#f], @racket[begin], @racket[quote],
+       form. If @racket[extend-stop-ids?] is not @racket[#f] then @racket[begin], @racket[quote],
```

Comment:
I agree that an explicit use of “then” here is clearer, but it’s still grammatically necessary to include the comma. Please use both.